Documentation/Signoffs/Sign.md

```diff
@@ -0,0 +1,180 @@
+# Sign System Detail Design
+
+
+
+## Function of the Subsystem
+
+The sign will be a physical entity at the entrance of the parking lot that will display the current number of available parking spots in the parking lot. This will allow students to decide if they even enter the parking lot and has the potential to save the students time searching for a parking spot. This subsystem is to be used in unity with the mobile application which will also display the same information. It will acquire its data from the server/database wirelessly with the initial data being from the data acquisition subsystem(s). It will then use custom seven-segment displays, a microprocessor, and a power transistor circuit to display this information on the physical sign.
+
+## Constraints
+
+1. Seven-Segment Display & Controls
+- Must have waterproof controllable display viewable in the daylight (At least 2000 millicandelas at a 60 degree viewing angle or 1.68 Lumens [12] [13]) and from the road capable of showing number of open parking spots 
```

Comment:
There's not a citation 12 or 13. I don't think that 1.68 lumens is sufficient. I did see the forum in the citations and I don't think that is an appropriate source of information from which to base a design decision.